src/rust-2024/rpit-lifetime-capture.md

```diff
@@ -1,12 +1,313 @@
-# RPIT lifetime capture
+# RPIT lifetime capture rules
 
 🚧 The 2024 Edition has not yet been released and hence this section is still "under construction".
 
-This feature is partially implemented, and not yet ready for testing.
-More information may be found in the tracking issue at <https://github.com/rust-lang/rust/issues/117587>.
+This chapter describes changes related to the **Lifetime Capture Rules 2024** introduced in [RFC 3498], including opaque type *precise capturing* introduced in [RFC 3617].
+
+[RFC 3498]: https://github.com/rust-lang/rfcs/pull/3498
+[RFC 3617]: https://github.com/rust-lang/rfcs/pull/3617
 
 ## Summary
 
+- In all editions, `use<..>` bounds allow for capturing only the lifetime parameters needed in RPIT opaque types.
+- If a `use<..>` bound is elided (i.e. not present) then the compiler automatically inserts one according to edition-specific rules.
+- In Rust 2024, *all* in-scope generic parameters, including lifetime parameters, are included in these automatically inserted `use<..>` bounds.
```

Comment:
I know that you can get a lot of mileage out of explaining this in terms of `use<..>`, but it also feels obtuse, and possibly less approachable given that people are more familiar with just calling it capturing.

I would probably not frame this in terms of an implicit `use<..>`, because that's literally not what happens in the compiler. Let's just call it capturing?

Reply:
I feel like capturing isn't that widely understood, and that people are going to rapidly start thinking of this in a syntactic way in terms of `use<..>`, which is why it was framed this way, but I take the point, and I've reframed it in terms of capturing.